docs/yaml/objects/compiler.yaml

```
@@ -283,6 +285,9 @@ methods:
     (defaults to -1024), `high` (defaults to 1024) and `guess` to
     specify max and min values for the search and the value to try
     first.
+    For C-like languages, the header `stddef.h` and `stdio.h` are included
+    implicitly for normal compilation, only `stddef.h` is included when
```

```suggestion
    implicitly for native compilation, only `stddef.h` is included when
```

same in other places.

---

done, thanks